sql/src/test/java/io/crate/analyze/relations/RelationSplitterTest.java

```diff
@@ -80,17 +93,14 @@ public void setUp() throws Exception {
         NestedReferenceResolver referenceResolver = new NestedReferenceResolver() {
             @Override
             public ReferenceImplementation<?> getImplementation(ReferenceInfo refInfo) {
-                return null;
+                throw new NotImplementedException();
```

Comment:
use UnsupportedOperationException.  NotImplementedException is internal sun stuff